JavaScript/Algorithms/kadane's_algo.js

```diff
@@ -0,0 +1,87 @@
+ 
+'use strict';
+
+process.stdin.resume();
+process.stdin.setEncoding('utf-8');
+
+let inpStr = '';
+let curr = 0;
+
+process.stdin.on('data', inputStdin => {
+    inpStr += inputStdin;
+});
+
+process.stdin.on('end', _ => {
+    inpStr = inpStr.trim().split('\n').map(string => {
+        return string.trim();
+    });
+    
+    master();    
+});
+
+function lineread() {
+    return inpStr[curr++];
+}
+
+function master() {
+    console.log("INPUT TEST CASES");
+    let t = parseInt(lineread());
+    for(let i=0;i<t;i++)
+    {   
+        console.log("INPUT NUMBER OF ELEMENTS");
+        let line = lineread().split(' ');
+        let ele = parseInt(line[0]);
+        line = lineread().split(' ').map(x=>parseInt(x));
+        console.log("INPUT ARRAY ELEMENTS");
+        let a = new Array(ele);
+        for(let k=0;k<ele;k++){
+            a[k]=line[k];
+        }
+        let Ob = new Kadanes();
+        let sol = Ob.kadanesalgo(a,ele);
+        if(sol===0)
+            sol=0;
+        console.log(sol);

```

Comment:
Display a message in print form as what are you showing output to the user

Reply:
done please check it